src/macaron/config/defaults.ini

```diff
@@ -159,8 +159,6 @@ jenkins =
     withMaven
     buildPlugin
     asfMavenTlpStdBuild
-    ./mvnw
```

Comment:
Is the reason why we remove these two lines is because `mvnw` and `mvn` has been covered by the `[builder.maven]` section in `defaults.ini`?
These two seems to be left over from an older version of Macaron.

Reply:
Yes, these are not needed anymore. They also hardcode the path, which is not desired.